keras/backend/cntk_backend.py

```diff
@@ -321,8 +330,9 @@ def random_binomial(shape, p=0.0, dtype=None, seed=None):
     size = 1
     for _ in shape:
         if _ is None:
-            raise ValueError('CNTK backend does not support '
-                             'random op on batch axis for now.')
+            raise ValueError('CNTK Backend: randomness op with '
+                             'shape \'%s\' is not supported now. '
+                             'Please remove the batch axis.' % shape)
```

Comment:
I don't entirely understand what the recommendation is `Please remove the batch axis`?